### ISSUE TITLE
restart service from container like systemd

### ISSUE THREAD

**Issue by liyimeng** [OpenRC/openrc#518]
In systemd, systemctl communicate with systemd via dbus. Therefor it is possible to restart service via calling "systemctl reload && systemctl restart myservice"  I am not sure if this is possible in openrc. If not  can this be a feature to add. 

I know a couple of popular cloud native open source project has to give OpenRC in favour of systemd  because of this reason.

**Comment by WhyNotHugo** [comment]
Does `rc-service $SERVICE restart` not work for you?

**Comment by liyimeng** [comment]
No, that is not working for me.  I was not clear enough to describe the problem, but If I have a look into the issue I refer above. It can be more clear. 

In systemd case, you can  mount dbus file descriptor into container, allow systemctl  control systemd service in host from inside of container. 
 
`rc-service $SERVICE restart` can only work when you run at host, not from inside a container.
